### ISSUE TITLE
Content: Update About Page

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#642]
We want to make the following updates to the /about page in preparation for the Phase 1 launch.

<h1>About</h1>

<h2>City of Chicago</h2>
Mayor Emanuel and the City of Chicago have made a firm commitment to providing children across the city with access to high-quality education, from early learning programs all the way up through college. Since taking office, Mayor Emanuel has invested over $45 million into expanding early childhood programs and building quality, access, and transparency throughout the city. Launched in 2012, the Early Learning Portal ensuring that parents have access to the information they need to make the choices that are best for their children and families.

_Note: Need partial Spanish translation_

<h2>Chicago Public Schools</h2>
Chicago Public Schools works to ensure that every student in every community has access to a high quality education that prepares them for success in college and career.

_Spanish translation is the same_
Las Escuelas Públicas de Chicago (Chicago Public Schools / CPS) trabajan para asegurar que todos los alumnos de todas las comunidades tengan acceso a educación de alta calidad que los prepare para desempeñarse con éxito en sus estudios universitarios y su carrera.

<h2>Chicago Department of Family</h2>
The Chicago Department of Family and Support Services (DFSS) is dedicated to supporting a continuum of coordinated services to enhance the lives of Chicago residents, particularly those most in need, from birth through the senior years.

_Spanish translation is the same_
El Departamento de Servicios Familiares y de Apoyo de Chicago (Chicago Department of Family & Support Services / DFSS) está dedicado al apoyo de una gama de servicios coordinados para mejorar la vida de los residentes de Chicago, sobre todo los más necesitados, desde el nacimiento hasta la edad avanzada.

New logo:
![dfss new](https://cloud.githubusercontent.com/assets/5550969/15872474/3d2a64f6-2cbf-11e6-8d73-486f9f01cbd3.jpg)

<h2>Smart Chicago Collaborative</h2>
The Smart Chicago Collaborative is a civic organization devoted to using technology to make lives better in Chicago. We work to apply the transformative power of technology to solve problems for the people of Chicago.

_Spanish translation is the same_
El Smart Chicago Collaborative es una organización cívica dedicada al uso de la tecnología para mejorar la vida de los residentes en Chicago. Trabajamos para aplicar el poder de transformación de la tecnología para resolver los problemas de las personas de esta ciudad.

--------------------

@theandrewbriggs Our partners are asking that we add additional partners as a list with an image of their logo. The name of the organization will link to the homepage of each org.

<h2>Other Partners</h2>

<h3>University of Chicago Urban Education Lab</h3>
https://uel.uchicago.edu/
![uel_logo](https://cloud.githubusercontent.com/assets/5550969/15872598/e73e6c26-2cbf-11e6-87fe-267b65c98b22.png)

<h3>Illinois Action for Children</h3>
http://www.actforchildren.org/
![iafc](https://cloud.githubusercontent.com/assets/5550969/15872626/fcfea512-2cbf-11e6-839b-4986bb60be7f.png)

<h3>The Ounce of Prevention Fund</h3>
http://www.theounce.org/
![theounce](https://cloud.githubusercontent.com/assets/5550969/15872644/13c26ee6-2cc0-11e6-9a98-3f5a722a631e.png)

<h3>Chicago Department of Public Health</h3>
http://www.cityofchicago.org/city/en/depts/cdph.html
![logo_partner_cdph](https://cloud.githubusercontent.com/assets/5550969/15872655/248234c8-2cc0-11e6-96b3-48b9e16b2fe0.png)

<h3>Metropolitan Family Services</h3>
https://www.metrofamily.org/
![metrofamily](https://cloud.githubusercontent.com/assets/5550969/15872659/288c137c-2cc0-11e6-99ed-cf47b2e0dd2c.gif)

<h3>EverThrive Illinois</h3>
http://www.everthriveil.org/
![everthrive il](https://cloud.githubusercontent.com/assets/5550969/15872694/4f00f766-2cc0-11e6-9ce2-29f8573b9e03.png)

**Comment by theandrewbriggs** [comment]
@smarziano - The UChicago UEL has its own main entry and description, just to clarify do we want to move that from the main partners list to the other partners like show above?

**Comment by smarziano** [comment]
We are moving UEL from the main partners section to the other partners section.

**Comment by smarziano** [comment]
Please add Erikson Institute (http://www.erikson.edu/) to the "Other Partners" section:

![erikson-institute](https://cloud.githubusercontent.com/assets/5550969/16098840/b6fd4da0-331a-11e6-90aa-7c12b82c6fbd.jpg)

**Comment by smarziano** [comment]
Erikson Institute has been added to staging. @theandrewbriggs Can we center this logo on the last line just so it reads better?

<img width="1023" alt="screen shot 2016-06-16 at 5 50 26 pm" src="https://cloud.githubusercontent.com/assets/5550969/16135972/8d4ea3be-33ec-11e6-949a-98ede262f86d.png">

**Comment by smarziano** [comment]
Centering updated with commit https://github.com/smartchicago/chicago-early-learning/commit/5585b05ac634d1dd3ceebebf41965a003a3a35d2

Waiting on merge

**Comment by smarziano** [comment]
This page looks great: https://chicagoearlylearning.org/city-resources

We still need Spanish translations. Will create new issue for post-launch milestone.
